app/theme/client/imports/components/popover.css

```diff
@@ -32,10 +32,12 @@
 	&__content {
 		position: absolute;
 
-		display: flex;
+		display: -webkit-box;
 
 		flex-direction: column;
 
+		height: 70%;
```

Comment:
When I have a tall screen/fewer buttons in the popover, doesn't this make it bigger than needed?

Reply:
Yes, it does. I have included a media query to adjust the height in case of taller screens. Hope that fixes the issue.

Reply:
We still have a problem, imagine I have another popover with only one button, it'd still be bigger than the content. Why don't you try max-height? Then (in theory) you won't need anything else, it'll only shrink when the popover is bigger than 70% of its parent.

Reply:
I have made the changes and it works correctly now. Thanks a lot for the pointers :)